source/synthDrivers/oneCore.py

```diff
@@ -81,6 +84,13 @@ def convertCharacterModeCommand(self, command):
 		# Therefore, we don't use it.
 		return None
 
+	def convertLangChangeCommand(self, command):
+		lcid = languageHandler.localeNameToWindowsLCID(command.lang)
+		if lcid in (0, languageHandler.LOCALE_CUSTOM_UNSPECIFIED):

```

Comment:
Honestly, I'm not sure. I did this because the other place we use that function checks for 0. It seems we can return 0 on Windows XP (where we implement all of that function ourselves), but that doesn't apply here, since this is always Windows 10. On Vista and later, we use the Windows [LocaleNameToLCID](https://msdn.microsoft.com/en-us/library/windows/desktop/dd318711(v=vs.85).aspx) function. That function says it can return 0 for invalid parameter values, but I don't know how to trigger this.

What do you think? On one hand, it might be better to remove the 0 so users notice potential unknown problems and report them. On the other hand, if we do get 0, there's probably nothing we can do and we'll end up just ignoring the language anyway.